services/proxy/pkg/command/server.go

```diff
@@ -272,6 +272,10 @@ func loadMiddlewares(ctx context.Context, logger log.Logger, cfg *config.Config,
 		})
 	}
 
+	authenticators = append(authenticators, middleware.AppAuthAuthenticator{
```

Comment:
I assume we want to enable this middleware, only if really needed? So we'd need a config knob for that.

Reply:
Until we have the details of all that sorted out I still think we should have the middleware disabled by default and only enabled if explicitly  wanted. (Isn't that also what was agreed on in the planning?)

Reply:
So if the service is not running by default, why should the middleware be enabled?

Reply:
Because of users want access to their calendars, address books and files via client which require username and password and do not support openid connect.

Refs #8357

Reply:
added a flag for it - see README.md